### PR TITLE
Suggested replies refactor

### LINKED ISSUE #24: Show a warning when deleting a suggested reply
This kind of action shouldn't happen very often - best to warn user against accidental deletion

### LINKED ISSUE #26: Show a character count for the suggested replies and a little warning if exceeding 160 chars

### PR DESCRIPTION
**Pull request by ktkization**
Closes #24 
Closes #26

### CODE REVIEW

File: webapp/web/configuration.css
Comment (marianamarasoiu) [inline webapp/web/configuration.css]: I think this would look better lighter in colour:
```suggestion
  background: rgba(0, 0, 0, 0.33);
```

**Review by marianamarasoiu — CHANGES_REQUESTED** [review]
thanks @ktkization - looks great! can you please also make the character count update when the response is being actively edited? (ie. on each key press) thanks!

**Comment by ktkization** [comment]
@marianamarasoiu character count now on input too. Also please merge this after #37 to avoid merge conflicts.

File: webapp/web/configuration.css
Comment (marianamarasoiu) [inline webapp/web/configuration.css]: single pixel was fine, this is now too bright. please revert here and below as well

Comment (marianamarasoiu) [reply webapp/web/configuration.css]: actually I see you're using it to show that the box is selected.. so let's keep the outline, but can we use a less red colour for both border and outline? maybe something like `salmon` or something closer to that rather than `#ff0000`?

**Review by marianamarasoiu — APPROVED** [review]
LGTM with a small nit, thanks!

File: webapp/web/configuration.css
Comment (ktkization) [reply webapp/web/configuration.css]: Changed also the counter to salmon when it errors

**Review by marianamarasoiu — APPROVED** [review]

**Pull request merged**
